content/en/_index.md

```diff
@@ -218,6 +302,18 @@ a file or a specific action is not.
         // Snippet content ...
     }
   {{< /tab >}}
+  {{< tab header="Node.js" >}}
+    function main(requiredArg) {
+      // This argument is required, and should be declared
+      const requiredArg = '...';
+
+      exampleSnippet(requiredArg)
+    }
+
+    const exampleSnippet = (requiredArg) => {
```

Comment:
nit: we usually don't use arrow functions.

Reply:
Done (except for a lambda on line `239`).